ocp.tf

```diff
@@ -21,18 +21,67 @@
 provider "libvirt" {
     uri = var.libvirt_uri
 }
-
 resource "random_id" "label" {
     count = var.cluster_id == "" ? 1 : 0
     byte_length = "2" # Since we use the hex, the word lenght would double
     prefix = "${var.cluster_id_prefix}-"
 }
 
+resource "random_id" "b" {
+    byte_length = "3"
+}
+resource "random_id" "m" {
+    count       = var.master["count"]
+    byte_length = "3"
+}
+resource "random_id" "w" {
+    count       = var.worker["count"]
+    byte_length = "3"
+}
+
 locals {
     # Generates cluster_id as combination of cluster_id_prefix + (random_id or user-defined cluster_id)
     cluster_id  = var.cluster_id == "" ? random_id.label[0].hex : "${var.cluster_id_prefix}-${var.cluster_id}"
+
+    bootstrap = {
+        ip      = cidrhost(var.network_cidr, 3)
+        mac     = format(
+                    "52:54:00:%s:%s:%s",
+                    substr(random_id.b.hex,0,2),
+                    substr(random_id.b.hex,2,2),
+                    substr(random_id.b.hex,4,2)
+                )
+    }
+    first_master_ip = 4
```

Comment:
should these vars be referred to as `octets` to reflect the correct usage ?

Reply:
Changed it to `first_master_hostnum` as per given in [docs](https://www.terraform.io/docs/configuration/functions/cidrhost.html).